app/webpacker/css/admin/v2/components/buttons.scss

```diff
@@ -21,7 +21,7 @@ button,
     @include backgroundAndBorder($v2-light-grey);
   }
 
-  &:not(.disabled):not([disabled]):not(.secondary) {
+  &:not(.disabled):not([disabled]):not(.secondary):not(.cancel) {
```

Comment:
Is it better to override the style for these selectors again instead of using `not`?

It's a bit old now but I remember that Google disallowed `not` for their fast loading AMP results saying that evaluating CSS would be much faster without `not` and without recursion. I'm not sure if we should care about that though.

Reply:
Oh, you are changing that already, cool.